proxy_agent/src/shared_state.rs

```diff
@@ -139,12 +193,33 @@ pub mod shared_state_wrapper {
 }
 
 /// wrapper functions for KeyKeeper related state fields
+/// Example:
+/// ```rust
+/// use proxy_agent::shared_state::key_keeper_wrapper;
+/// use proxy_agent::shared_state::SharedState;
+/// use std::sync::{Arc, Mutex};
+///
+/// let shared_state = SharedState::new();
+///
+/// // set the key once the MSP feature is enabled
```

Comment:
maybe define MSP feature here for clarity

Reply:
removed msp